components/post_profile_picture/post_profile_picture.tsx

```diff
@@ -112,12 +114,11 @@ export default class PostProfilePicture extends React.PureComponent {
                 profileSrc={profileSrc}
                 isEmoji={isEmoji}
                 status={status}
-                userId={user ? user.id : null}
+                userId={user ? user.id : undefined}
```

Comment:
Could we use `user.?id` here instead?

Reply:
Hi, thank you for the feedback. I already change it to  `user?.id`